[2024-10-16T00:51:10.442+0000] {taskinstance.py:1159} INFO - Dependencies not met for <TaskInstance: group_dag.transforms manual__2024-10-15T10:09:30.567294+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2024-10-16T00:51:10.444+0000] {local_task_job.py:103} INFO - Task is not able to be run
